README.md

```diff
@@ -43,9 +43,41 @@ npx web-ext run --verbose --firefox-binary /path/to/nightly/firefox -s extension
 
 # Release
 
-1. Bump version in ``package.json``, ``update.json``, and ``extension/manifest.json``
-2. Tag commit ``git tag -a X.Y.Z`` and push it
-3. Create release with changelog on Github
-4. Check that ``FirefoxCI`` action has run for tagged commit
-5. Request sign-off on Slack channel ``#addons-pipeline``
-6. Download signed build from Task Cluster, and attach ``remote-settings-devtools-X.Y.Z.xpi`` binary file on Github release page
+### Prerequisites (get access to Ship-It)
+
+1. Create a ticket to be added to the VPN group (can clone and edit [this Bugzilla ticket](https://bugzilla.mozilla.org/show_bug.cgi?id=1740098))
+2. Ask in the [#addons-pipeline](https://mozilla.slack.com/archives/CMKP7NPKN) channel to be added to the `XPI_PRIVILEGED_BUILD_GROUP` to get access to create an XPI release for `remote-settings-devtools` on [Ship-It](https://shipit.mozilla-releng.net/)
+
+### Create a new tag/release
+
+1. Bump version in `package.json`, `update.json` (has **three** version instances that need to be updated), and `extension/manifest.json`
+2. Tag commit `git tag -a X.Y.Z` and push it `git push origin X.Y.Z`
```

Comment:
I wonder if it would be worth abstracting this into a little convenience script so that instead of having to remember to update in 5 places, tag, then push, we run like `scripts/generate_release.sh` or something that. Not necessary for this PR, but if we find ourselves forgetting to update these values, maybe something we consider.

Reply:
Yeah I like that! Just created https://github.com/mozilla-extensions/remote-settings-devtools/issues/85